### PR TITLE
add favicons for all devices

### PR DESCRIPTION
**Pull request by Arhell**
/kind feature

**Which issue(s) this PR fixes**:
resolve https://github.com/karmada-io/website/issues/60

### CODE REVIEW

File: src/plugins/faviconCustomPlugin.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                        attributes: {
+                            name: "msapplication-config",
+                            content: "/favicons/browserconfig.xml",
+                        },
+                    },
+                ],
+            };
+        },
+    };
+};
```
Comment (RainbowMango) [inline src/plugins/faviconCustomPlugin.js:73]: No new line at end of file.

Comment (Arhell) [reply src/plugins/faviconCustomPlugin.js]: Done

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**